Makefile

```diff
@@ -67,155 +67,158 @@ test-qe-black-box: build-qe
 ###########################
 
 all-dbs-up:
-	docker compose -f docker-compose.yml up -d --remove-orphans
+	docker-compose -f docker-compose.yml up -d --remove-orphans
```

Comment:
That should be equivalent on modern Docker installations. `docker compose` is actually preferred I think.

Reply:
`docker compose` doesn't work with podman, so I have to make this change on every branch. This is always reverted before I submit for review, and this branch is far from that point.

Reply:
This PR is not anywhere close to ready for review.

Reply:
> `docker compose` doesn't work with podman

I switched from podman to docker in rootless mode mainly because of this

Reply:
See docs for more info
https://docs.docker.com/compose/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2
`docker compose` = Compose v1 = legacy
`docker-compose` = Compose v2 = recommended

Reply:
🤔 Isn't it the other way around? @Jolg42

<img width="723" alt="Screenshot 2023-09-15 at 12 43 08" src="https://github.com/prisma/prisma-engines/assets/210307/69b8e820-baab-431a-8c6e-21dbcaf2387d">

Reply:
It's the other way around @Jolg42, `docker-compose` is a legacy standalone tool written in Python that's not maintained anymore but still useful for e.g. podman (as podman-compose is still so early in development it's barely useful in practice yet). `docker compose` is the new one rewritten in Go as a Docker CLI plugin.

On modern Docker Desktop installations they are actually the same thing as `docker-compose` is now just an alias for `docker compose`, but on Linux, if you install Docker from your distro repositories, they are typically different.

Reply:
ah sorry, your last comment didn't load for me for some reason, I was too late here